src/apps/content-editor/src/app/components/Editor/Field/Field.tsx

```diff
@@ -283,10 +283,16 @@ export const Field = ({
       return (
         <AIFieldShell
           ZUID={fieldData?.ZUID}
-          name={fieldData?.name}
```

Comment:
This change strike me as odd. How does this relate to blocks? How is the `name` variable available? If it was already available, why add it now as an or condition?

Reply:
Same questions for label

Reply:
I'm seeing that this component is being used in the blocks app. I guess I'm not sure why there is a difference in the props.